addons/l10n_gcc_invoice/views/report_invoice.xml

```diff
@@ -344,33 +350,55 @@
                         <div class="col-6">
                             <table class="table table-sm" style="page-break-inside: avoid;">
                                 <tr class="border-black o_subtotal">
-                                    <td>
+                                    <td class="text-end">
+                                        <span t-field="o.amount_untaxed"/>
+                                    </td>
+                                    <td class="text-end">
                                         <strong>
                                             Subtotal
                                             /
                                             الإجمالي الفرعي
                                         </strong>
                                     </td>
-                                    <td class="text-end">
-                                        <span t-field="o.amount_untaxed"/>
-                                    </td>
                                 </tr>
                                 <t t-set="tax_totals" t-value="o.tax_totals"/>
                                 <t t-foreach="tax_totals['subtotals']" t-as="subtotal">
                                     <t t-set="subtotal_to_show" t-value="subtotal['name']"/>
-                                    <t t-call="account.tax_groups_totals"/>
+                                    <!-- copy-pasted template "account.tax_groups_totals" with reversed columns order -->
+                                    <t t-foreach="tax_totals['groups_by_subtotal'][subtotal_to_show]" t-as="amount_by_group">
+                                        <tr>
```

Comment:
Still checking exactly how to do it properly but to reverse a table, we just need to do this:
`<table style="direction: rtl;">`

Reply:
And I think we should do it that way because copying the template is going to break the localization as soon as we change the way the tax_totals is working

Reply:
I tried with `style="direction: rtl;"`, but it didn't work. I think I put it on `<tr>` level. I'm going to try to do it for `<table>`. Thanks for the hint

Reply:
Just going through my tab to close them and I realise I hadn't posted this:

I was able to directly use `document_tax_totals` and have some "ok tier" results. (I was losing the arab writings but I expect people to update translations on transifex at somepoint)

If it requires some fine tweaking, we could probably just do some inherit with `primary="True"` on it and call the inherited one.
If it's too many tweaking, it's better not inheriting it and just do the stuff in place.